### PR TITLE
Remove bootstrapping and CMF testing dependency

### PR DESCRIPTION
**Pull request by dantleech**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no

#### What's in this PR?

This PR removes all `bootstrap.php` and `console` files in bundles, and does not use the `symfony-cmf/testing` console.

#### Why?

The `bootstrap.php` files were repeated for each bundle, the `symfony-cmf` testing component was only being used for its `console` script.

We also remove the `symfony-cmf/testing` dependency.

#### Next step

Removing all the bundle `AppKernel` classes and configurations would be nice. Each test should create (via. a utility method) a kernel with whatever kernels and configuration the *test* requires. But this is not so important.

### CODE REVIEW

File: src/Sulu/Bundle/AdminBundle/phpunit.xml.dist [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<phpunit bootstrap="./Tests/bootstrap.php" colors="true">
+<phpunit bootstrap="../TestBundle/Resources/app/bootstrap.php" colors="true">
```
Comment (dantleech) [inline src/Sulu/Bundle/AdminBundle/phpunit.xml.dist]: If we were to make a subtree split then we would reference the testBundle from within the `vendor` directory.

**Comment by danrot** [comment]
@dantleech Is that something you are still working on?

**Comment by wachterjohannes** [comment]
@dantleech this PR would be a very good improvement because it removes the dependency to symfony-cmf/testing and it is not far away from working. so it would be cool to finish it

**Comment by danrot** [comment]
Yeah, from what I have seen it looks like only the SuluWebsocketBundle is missing.

**Comment by dantleech** [comment]
@danrot updated, it was just a bad path.

**Comment by wachterjohannes** [comment]
@dantleech can you also rebase it?

**Pull request merged**